azure/examples/express/package.json

```diff
@@ -21,5 +19,7 @@
         "typescript": "^3.0.3"
     },
     "peerDependencies": {
+        "@pulumi/cloud": "latest",
```

Comment:
/cc @CyrusNajmabadi this is in line with how all the other examples work (where we use peer-dependencies for packages in the same repository, since we link them when we run tests).